linter/lints/cabf_br/lint_crl_validity_period.go

```diff
@@ -3,21 +3,53 @@ package cabfbr
 import (
 	"time"
 
-	"github.com/letsencrypt/boulder/crl/crl_x509"
+	"github.com/zmap/zcrypto/x509"
 	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
 )
 
-// hasAcceptableValidity checks Baseline Requirements, Section 4.9.7:
-// The value of the nextUpdate field MUST NOT be more than ten days beyond the
-// value of the thisUpdate field.
-func hasAcceptableValidity(crl *crl_x509.RevocationList) *lint.LintResult {
-	validity := crl.NextUpdate.Sub(crl.ThisUpdate)
+type crlValidityPeriod struct{}
+
+/************************************************
+Baseline Requirements, Section 4.9.7:
+For the status of Subscriber Certificates [...] the value of the nextUpdate
+field MUST NOT be more than ten days beyond the value of the thisUpdate field.
+
+Although the validity period for CRLs covering the status of Subordinate CA
+certificates is longer (up to 12 months), Boulder does not produce such CRLs,
```

Comment:
Technically Boulder (via cmd/ceremony) does produce Subordinate CA CRLs; I think we just don't lint them yet.